bridges/TwitterBridge.php

```diff
@@ -475,11 +475,9 @@ public function collectData(){
 			$this->items[] = $item;
 		}
 
-		usort($this->items, array('TwitterBridge', 'compareTweetId'));
-	}
-
-	private static function compareTweetId($tweet1, $tweet2) {
-		return (intval($tweet1['id']) < intval($tweet2['id']) ? 1 : -1);
+		usort($this->items, function ($tweet1, $tweet2) {
+			return (intval($tweet1['id']) < intval($tweet2['id']) ? 1 : -1);
```

Comment:
How does this relate to the changes?

Reply:
This line `usort($this->items, array('TwitterBridge', 'compareTweetId'));` caused an error that the class is already defined. Not sure how that works out but it did give that error.